waterlog/src/reducers/leakLitresReducer.js

```diff
@@ -0,0 +1,34 @@
+import { FETCH_LEAK_LITRES_BEGIN, FETCH_LEAK_LITRES_SUCCESS, FETCH_LEAK_LITRES_FAILURE } from '../actions/Types';
+
+const initialState = {
+  item: {},
+  loading: false,
+  error: null
+};
+
+export default function (state = initialState, action) {
+  switch (action.type) {
+    case FETCH_LEAK_LITRES_BEGIN:
+      return {
+        ...state,
+        loading: true,
+        error: null
+
+      };
+    case FETCH_LEAK_LITRES_SUCCESS:
+      return {
+        ...state,
+        loading: false,
+        item: action.payload.litres
```

Comment:
remember to reset error to null on success

Reply:
done